mne/time_frequency/tfr.py

```diff
@@ -742,9 +745,26 @@ def plot_topo(self, picks=None, baseline=None, mode='mean', tmin=None,
         border : str
             matplotlib borders style to be used for each sensor plot.
         fig_facecolor : str | obj
-            The figure face color. Defaults to black.
-        font_color: str | obj
-            The color of tick labels in the colorbar. Defaults to white.
+            The figure face color. Defaults to black. If "black", spines are
+            white (else they are black).
+        font_color : str | obj
+            The color of text in the colorbar and title. Defaults to
+            spine_color (which is automatically set based on fig_facecolor
+            if None).
+        external_legend : bool | float
+            Plot an external x and y axis legend (usually time/unit), showing
+            and labelling time and unit ticks. Requires `evokeds` to hold
+            only 1 channel type. If float, an external legend is plotted and
+            scaled by this factor.
+        internal_legend : bool
+            Plot time and unit labels for individual channel subplots. If
+            external_legend is true, only tick labels, not axis labels are
+            plotted.
+        plot_ch_names : bool
```

Comment:
I think this is called `show_names` elsewhere. Have a look here: https://github.com/mne-tools/mne-python/search?utf8=%E2%9C%93&q=show_names